scripts/environment.sh

```diff
@@ -71,7 +71,6 @@ default_environment() {
 
     ns_arb_arch=native
     ns_arb_with_gpu=OFF
-    ns_arb_gpu=none
```

Comment:
How about keeping this variable, and allowing it to be set to `none`, `cuda`, `hip` or `clang-cuda`, and passing it directly to CMake:
```
cmake -DARB_GPU=$ns_arb_gpu
```

Reply:
I decided on using `ns_arb_with_gpu` for this purpose.